lms/static/sass/views/_program-details.scss

```diff
@@ -655,6 +617,62 @@
   }
 }
 
+
+// CSS used in both the main program detail and the program detail sidebar
+.btn-view-detail {
+  background: palette(primary, dark);
+  border-color: palette(primary, dark);
+  height: 37px;
+  width: auto;
+  border-radius: 0;
+  text-align: center;
+  font-size: 0.9375em;
```

Comment:
do we have any other heading styles or typography defaults from other buttons in the page?

Reply:
The only other one I'm seeing is a generic button class at https://github.com/edx/edx-platform/blob/5cb1af30dc13491e02b22e87e78c784957e14dc7/lms/static/sass/views/_program-details.scss#L308 but I'm not sure if it's worth pulling that out